BridgeCareApp/BridgeCare/Controllers/InvestmentStrategiesController.cs

```diff
@@ -22,8 +22,29 @@ public InvestmentStrategiesController(IInvestmentStrategies investmetStrategiesR
             db = context ?? throw new ArgumentNullException(nameof(context));
         }
 
-        // Get: api/InvestmentStrategies
+        ///<summary> Get: api/InvestmentStrategies
+        ///argument: NetworkModel
+        ///</summary>
         [ModelValidation("Given network data is not valid")]
         public IQueryable<InvestmentStrategyModel> Get(NetworkModel network) => investmentStrategies.GetInvestmentStrategies(network, db);
+
+        ///<summary> Post: api/InvestmentStrategies
+        ///argument: InvestmentStrategyModel
+        ///return : 200 sucess
+        ///         400 for bad input argument
+        ///         500 internal server error (uncaught exception)
+        ///</summary>
+        [ModelValidation("Given investment strategy data is not valid")]
+        public IHttpActionResult Post(InvestmentStrategyModel data)
```

Comment:
The controller class in our framework determines the name, by default it is created by removing the word 'controller'. The class is called: InvestmentStrategiesController. Mostly the code rolls with this to the point I have not looked for the way to override this behavior. 

technically it is probably what you say: http://localhost:50696/api/investmentstrategies/<post or get>, lots of testing in postman, but in there a post or get method, and corresponding tab is created. Should I change the 'use' string in the comment?, if so it will need to be changed in all the usage comments thus far from Saransh and myself.

Reply:
I checked,I can override naming. api/SaveInvestmentStrategies it  is